Makefile

```diff
@@ -14,19 +21,14 @@ endif
 
 all: manager
 
-# Create a local docker registry, start kinD cluster, and install Orkestra
+# Create a local docker registry, start kind cluster, and install Orkestra
 dev:
-	bash scripts/kind-with-registry.sh
+	make kind-create
 	helm upgrade --install orkestra chart/orkestra --wait --atomic -n orkestra --create-namespace --values ${CI_VALUES}
 
 debug: dev
```

Comment:
Running this target after running `dev` throws an error since the cluster already exists.

Reply:
```
~/github/azure/orkestra make debug           pr/helayoty/302→helayoty/helayoty/299-makefile-update ●
make kind-create
./scripts/create-kind-cluster.sh
ERROR: failed to create cluster: node(s) already exist for a cluster with the name "orkestra"
make[1]: *** [kind-create] Error 1
make: *** [dev] Error 2
```